docs/wiki/The library - Module usage.md

```diff
@@ -5,12 +5,8 @@ This section provides a guideline on how to use the CARML Bicep modules.
 ### _Navigation_
 
 - [Deploy template](#deploy-template)
-  - [Deploy local template](#deploy-local-template)
-    - [**Local:** PowerShell](#local-powershell)
-    - [**Local:** Azure CLI](#local-azure-cli)
-  - [Deploy remote template](#deploy-remote-template)
-    - [**Remote:** PowerShell](#remote-powershell)
-    - [**Remote:** Azure CLI](#remote-azure-cli)
+  - [PowerShell](#powershell)
+  - [Azure CLI](#azure-cli)
```

Comment:
Changes related to a different PR. Should be removed

Reply:
Agreed. Not sure how it got in here

Reply:
Removed